### PR TITLE
qemu.tests: wait for device ready when check inside guest

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

ID:1181031

### CODE REVIEW

**Comment by xutian** [comment]
[root@dhcp-8-209 qemu]# ../../../../../../tools/scan_results.py 
Test                                                                                                                                                                                        Status     Seconds  Info
----                                                                                                                                                                                        ------     -------  ----
        (Result file: /root/staf-kvm-devel/autotest-devel/client/results/default/status)
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.spice.nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.1.x86_64.io-github-autotest-qemu.usb.usb_boot.usb_kbd.with_usb_hub.one_usb_dev.uhci    GOOD       70       completed successfully
----                                                                                                                                                                                        GOOD       78

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by PyLearner** [comment]
Acked-by: Meng Yang meyang@redhat.com

File: qemu/tests/usb_basic_check.py [hunk 1]
```diff
@@ -20,6 +21,22 @@ def check_usb_device_monitor(test, params, env):
                              "returns: \n%s" % (params.get("product"), o))
 
 
+def check_usb_device_guest(session, item, cmd):
+    def __check(session, item, cmd):
+        output = session.cmd_output(cmd)
```
Comment (ypu) [inline qemu/tests/usb_basic_check.py]: Here seems also has the problem the the timeout in wait_for is shorter then the timeout inside.

**Comment by PyLearner** [comment]
Acked-by : Meng Yang meyang@redhat.com

**Comment by xutian** [comment]
thanks @suqinhuang @PyLearner , merged

**Pull request merged**